core/controllers/suggestion.py

```diff
@@ -264,17 +264,86 @@ def post(self) -> None:
         # adding an image, there is no method to remove the uploaded image.
         # See more - https://github.com/oppia/oppia/issues/14298
         if suggestion_type != feconf.SUGGESTION_TYPE_ADD_QUESTION:
+            assert isinstance(
+                suggestion,
+                suggestion_registry.SuggestionTranslateContent
+            )
             files = self.normalized_payload.get('files')
             new_image_filenames = (
                 suggestion.get_new_image_filenames_added_in_suggestion()
             )
             if new_image_filenames and files is not None:
-                _upload_suggestion_images(
+                self._upload_new_images_in_translation(
                     files, suggestion, new_image_filenames
                 )
 
+            self._copy_images_from_target_exploration_to_translation(suggestion)
```

Comment:
I'm a bit confused ... why do we upload and then copy? I.e. why isn't adding the images that were part of the translation suggestion into the suggestion's directory sufficient?

I would have thought that we would: 

(a) add the images that were sent as image data into the suggestion's directory
(b) copy the images that were part of the translation suggestion's text, but that were not sent as image data, into the suggestion's directory

Reply:
@seanlip For context, the copy tool creates an image in the HTML string, whose `filepath` points to the original image's filepath. The original image is an image in a card's content belonging to the target exploration. The intent of this algorithm isn't to upload images created using the copy tool. Instead, we are trying to copy the original images for the translation suggestion to use as assets. Since the images created using the copy tool have the same `filename` as the original image, this will also allow the images created using the copy tool to load when reviewing the translation. In short, we save images uploaded directly from the translation submitter, and then we copy the original images within the target exploration's card's content for the suggestion to use later on.

I think the highlighted method name is not clear about this. I think the name `_copy_images_in_target_exploration_card_to_translation(suggestion)` would be better since it better explains what images I'm referring to.

Reply:
Ah yup, as long as we just use the images in the translation suggestion's card, then that seems fine. I just want to ensure we aren't copying "extra" images that aren't part of the card.

Reply:
I renamed the method responsible for copying the images in the target exploration content.